Bash_scripts/HW24/run_ansible.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+########################## REQUIREMENTS
+# sudo apt-get install ansible unzip
+########################## VARIBLES
+
+PLAYBOOK_GZ="playbook.zip"
+PLAYBOOK_FILE="hw24.yml"
+INVENTORY_FILE="hosts.yml"
+INVENTORY_FILE_2="hosts_2.yml"
+EXTRA_ARGS="--list-tasks --list-hosts"
+
+WORK_FOLDER=from_hw23
+SOURCE_DIR=$(pwd)
+DESTINATION_DIR=/tmp/update
+LOGFILE=$SOURCE_DIR/update.log
+
+
+########################## CODE
+
+echo "Starting log" >> $LOGFILE
+
+echo -n "Adding key... " >> $LOGFILE
+cd ~/.ssh
+cat id_rsa.pub >> authorized_keys
+SORTIDA=$?
+echo OK >> $LOGFILE
+
+echo -n "Cleaning folders... " >> $LOGFILE
+rm -rf $DESTINATION_DIR 2> /dev/null
+mkdir $DESTINATION_DIR
+cp -r $SOURCE_DIR/$WORK_FOLDER $DESTINATION_DIR
+SORTIDA=$?
+echo OK >> $LOGFILE
+
+# echo -n "Uncompressing update... " >> $LOGFILE
+# cd $DESTINATION_DIR
+# unzip $PLAYBOOK_GZ
+# rm $PLAYBOOK_GZ
+# SORTIDA=$?
+# echo OK >> $LOGFILE
+
+echo "Start Ansible... " >> $LOGFILE
+cd $DESTINATION_DIR/$WORK_FOLDER
+ansible-playbook $PLAYBOOK_FILE -i $INVENTORY_FILE -i $INVENTORY_FILE_2 >> $LOGFILE
```

Comment:
This will be broken for only one one inventory file

Reply:
Yes, I fixed it. I changed file hosts.yml and chenged run_ansible.sh
![hw26t20](https://user-images.githubusercontent.com/78340773/118182198-611b8700-b441-11eb-9434-48ae9e405e0c.png)